packages/graphql-mesh-server/lib/maintenance.ts

```diff
@@ -72,10 +72,8 @@ export class Maintenance extends Construct {
       resources: [efsVolume.fileSystemArn, accessPoint.accessPointArn],
     });
 
-    efsVolumeSecGroup.addIngressRule(
-      Peer.anyIpv4(), // Can't get the IP address of each container as we don't know them at deploy time!
-      Port.tcp(2049),
-      "File access"
+    efsVolume.connections.allowDefaultPortFrom(
+      props.fargateService.connections
```

Comment:
I could not work this out when I wrote this!!! Thank you for fixing it :pray:

Reply:
Actually, I think both the task and the function need to access the filesystem right? In that case it might be better to allow the VPC CIDR? (instead of Any IPv4)

Reply:
Or, create security groups for Lambdas and attach the SG to the ingress rule :thinking: what do you think?

Reply:
I've made it so that the vpc ipv4 addresses are allowed